### DISCUSSION TITLE
Base Ratio set to zero?

### DISCUSSION THREAD

**Discussion by Jordain** [hako-mikan/sd-webui-regional-prompter#149] [Q&A]
Can the Base Ratio be set to zero? Can I prevent the base ration to have any influence on the subsequent regions? I tried setting it to zero, but it seems to still have an affect on the other regions.

**Comment by Symbiomatrix** [comment]
Depends on the mode. In inpaint+, it is designed to be possible: base mode has a weighted effect on coloured regions, and 100% effect on noncoloured regions, so it's perfect for filling in background at 0%. In matrix, 0% base should in theory have no effect, beyond creating one more context set. Prompt, haven't the foggiest what base does.

**Reply by Jordain** [reply]
What I am trying to do is use a character image as a reference in control net and have a matrix (for example: 2x2) of facial expressions. I sort of get the results, but the issue is that with every next section in the matrix it becomes a deeper and deeper version of the prompt. The first expression is perfect then the 2nd expression gets the next prompt but part of the first prompt as well, so that's why I was trying to set the Base Ratio to zero, but it doesn't appear to work.

**Reply by Symbiomatrix** [reply]
I don't quite follow what you're trying to do or how base is related.
I take it that the reference image supplied to controlnet is a 2x2 grid of facial expressions (say, as sketch or depthmap variants).
In that case, logically you would use matrix mode with divide ratio 1,1,1;1,1,1 (a square divided into 4 quadrants), and write 4 prompts, each describing a section. They sound completely separate so there is no need for base (and common is useful only if it's the same character). There should be no concept bleed unless we're talking loras (in which case latent is preferable; possibly so in general because attention tends towards cohesion in general, which is undesirable in this case), or compatibility with controlnet has deteriorated.

**Reply by Jordain** [reply]
I tested it again without the lora and got it to work as intended with the 1 x 2 grid, i'll try a 2 x 2 grid later. The bleed happens with the Lora. I'll read the documentation again with the Loras because it seems like there are issues or you need special parameters with them.

**Reply by Symbiomatrix** [reply]
Alright, good.
That is correct. Even with high steps + low cfg and neg parameters, they can be hard to control. Moreso in the backend.
